result.go

```diff
@@ -54,11 +56,11 @@ type Config struct {
 // gateways, and routes assigned to sandbox and/or host interfaces.
 // c) DNS information. Dictionary that includes DNS information for nameservers,
 // domain, search domains and options.
-func (c *libcni) GetCNIResultFromResults(results []*current.Result) (*CNIResult, error) {
+func (c *libcni) GetCNIResultFromResults(results []*current.Result) (*Result, error) {
```

Comment:
Should this be called `GetResultFromResults`? 🤔 (sounds confusing though)

Reply:
Hm... yes, naming is a bit confusing overall. It doesn't seem to be [part of the interface, so we could rename](https://github.com/containerd/go-cni/blob/8fbf3637eb5f67bb16eaacd4ec23a82e4d33b3ec/cni.go#L31-L42)

 Actually looking now;

- if it needs to be exported(?) it looks like it's [only used in `CNI.Setup()`](https://github.com/containerd/go-cni/blob/8fbf3637eb5f67bb16eaacd4ec23a82e4d33b3ec/cni.go#L151-L159), but not anywhere else (checked in containerd code)
- currently it takes `c *libcni` as a receiver, but only doing so for locking; Looking at the `Setup()` code again, and if we're ok with un-exporting, we could move the `c.Networks()` into the function, and change it to (e.g.) just

```go
func(c *libcni) results() (*Result, error) {
```

Reply:
I'll have a look at making those changes (will push as a second commit)